cytominer_database/ingest_incl_parquet.py

```diff
@@ -0,0 +1,203 @@
+# May 4th, TODO:
+# Main: Clean up (remaiming: tableSchema.py). Build base class and engine-extensions.
+# Other: - get rid of superfluous arguments. 
+# ---> skip_table/image_prefix
+    # alternative : instead of passing "skip_image_prefix", we could load the setting from the config file. Needs adjustments of tests.
+    # Currently: prefixes are never skipped for compartments.
+    # They are skipped for images by default, but can be added if skip_image_prefix is passed as True.
+# ----> write a new validate_csv_set() and deal with weird image/compartment output structure         
+"""
```

Comment:
Loving this documentation!! I am wondering if you thought about moving some of this text to the wiki in this repo

Reply:
Thanks :D . Yes, that was the long-term plan. For now I removed the additional docstring (saved the text, though) - will probably just add the corrected version to the Wiki?

Reply:
sounds good - as long as this valuable documentation is not lost and there is a plan!